Add installation guide for Dawarich on Unraid

Pa7rickStar opened the pull request:
This pull request adds a comprehensive installation guide for setting up Dawarich on Unraid.

**Important notice:** This guide utilizes two new Community Applications (CA) templates for Unraid (for PostGIS and redis databases), created by Pa7rickStar. These Applications are not listed in the Unraid Community Applications store yet, because they are still under review.
Due to the vast experience the [maintainer](https://github.com/nwithan8) of the other two CA templates it would make sense to ask him to review this guide before merging.

**New Dawarich Unraid Installation Documentation:**

* **Step-by-step guide for installing Dawarich on Unraid:** Details the setup process for all required containers (`dawarich_db`, `dawarich_redis`, `dawarich_app`, `dawarich_sidekiq`), including configuration recommendations.
* **Instructions for creating and preserving a user-defined Docker network:** Explains how to set up a bridge network for container isolation and how to configure Unraid to preserve custom networks.
* **Translating Dawarich docker-compose file to Unraid templates:** Includes commands and explanations for checking container health and the custom network which is often omitted in Unraid setups.
* **Support for Unraid related questions:** Provides links to relevant forums and repositories for community support, as the Dawarich developer does not maintain Unraid templates.

nwithan8 commented on docs/How_to_install_Dawarich_on_Unraid.md:
```diff
@@ -0,0 +1,200 @@
+
+There is an official [PostGIS](https://hub.docker.com/r/postgis/postgis) CA you can use for `dawarich_db` and an official [redis](https://forums.unraid.net/topic/89502-support-a75g-repo/) CA you can use for `dawarich_redis`. However, if you don’t want to set up the correct volume paths, environment variables, health-checks and arguments by yourself, [Pa7rickStar](https://github.com/pa7rickstar) h ...
+
+For [`dawarich_app`](https://github.com/nwithan8/unraid_templates/blob/main/templates/dawarich.xml) and [`dawarich_sidekiq`](https://github.com/nwithan8/unraid_templates/blob/main/templates/dawarich_sidekiq.xml) [grtgbln](https://forums.unraid.net/profile/81372-grtgbln/) aka. [nwithan8 on Github](https://github.com/nwithan8) is [maintaining](https://github.com/Freika/dawarich/issues/928#issuecomm ...
```
```suggestion
For [`dawarich_app`](https://github.com/nwithan8/unraid_templates/discussions/273) and [`dawarich_sidekiq`](https://github.com/nwithan8/unraid_templates/discussions/310) [grtgbln](https://forums.unraid.net/profile/81372-grtgbln/) aka. [nwithan8 on Github](https://github.com/nwithan8) is [maintaining](https://github.com/Freika/dawarich/issues/928#issuecomment-2749287192) Unraid CA templates.
```

Updated links to point to related Discussions threads.

nwithan8 reviewed (commented):
LGTM, very detailed!

Pa7rickStar commented:
- Fixed: wording of a few sentences.
- Fixed: Broken link to unraid docs.
- Implemented suggestions from @nwithan8.
- Added: Privacy warnings due to the default values for `PHOTON_API_HOST` in the `dawarich_app` and `dawarich_sidekiq` Unraid CA templates.
- Added: Guide on how to setup the Photon CA I created for reverse geocoding.

@Freika My repository for the Unraid Community AppStore is finally approved and the containers are online. This guide can go live 🥳

Beste Grüße aus Bayern!

Freika commented:
Hey @Pa7rickStar, can I ask you to create a topic in our forum instead? It would be easier to update and discuss there: https://discourse.dawarich.app/c/community-guides/8

Thank you!

Freika commented:
As an option, you can create a topic on our forum with the link to the one on Unraid forums that is being actively updated and maintained, totally fine by me.

My personal problem with community guides making it to the official docs is that I'll have to maintain it. 

If I have the expertise, I'll have to "merely" make a proper review of changes, maybe try to reproduce them. Considering I have little time to spare, guides won't be reviewed as fast as they should be.

If I don't have the expertise, like with Unraid, I'll have to merge the PR blindly, and people will rely on it as if I checked it and it worked, even though it's not the case.

Community guides posted by their authors on the forum are lifting this unnecessary responsibility from me.

Another thing is that the forum should become a place for non non-tech-savvy community as well, i.e. Cloud users. There is no way people who want to just use the product will register on Github to get help. Our Discord community proved to be very useful, so extending it to the forum feels only natural to me, and it also opens it for indexing by search engines.

The forum works with markdown well, so you can either post the guide on the forum, or, if you see it more convenient and making more sense, just create a topic there with the link, as I suggested above.

And also — thank you!
